play-json/jvm/src/test/scala/play/api/libs/json/ReadsSpec.scala

```diff
@@ -224,6 +224,12 @@ class ReadsSpec extends org.specs2.mutable.Specification {
       }
     }
 
+    "not be read from invalid number" in {
```

Comment:
Can we also have tests for `java.util.Date`, `java.sql.Date` and other affected?

Reply:
added more tests